app/controllers/concerns/two_factor_authenticatable_methods.rb

```diff
@@ -119,7 +119,7 @@ def handle_remember_device_preference(remember_device_preference)
   def increment_mfa_selection_attempt_count(auth_method)
     user_session[:mfa_attempts] ||= {}
     user_session[:mfa_attempts][:attempts] ||= 0
-    if user_session[:mfa_attempts][:auth_method] != auth_method
+    if user_session[:mfa_attempts][:auth_method].to_s != auth_method.to_s
```

Comment:
webauthn gets converted to a string when it's added to the session
[2] pry(#<Users::WebauthnSetupController>)> auth_method
=> :webauthn.1    | 
[3] pry(#<Users::WebauthnSetupController>)> user_session[:mfa_attempts]

it would always set the increment to zero because the 2 values were coming out inequal

Reply:
Thinking it's because the auth method may or may not be set to "_platform" 🤔

Reply:
Yes, that's a good call-out. The WebAuthn controller handles both Security Key and Face or Touch Unlock, and we probably want to differentiate with `webauthn_platform` vs. `webauthn`.